### DISCUSSION TITLE
Animated GIFs in READMEs only loop once

### DISCUSSION THREAD

**Discussion by adamlui** [community/community#47709] [Repositories] [answered]
### The problem

For example, I added an animated GIF to the Installation section of https://github.com/adamlui/userscripts/blob/master/chatgpt/duckduckgpt/README.md but it stops after one loop

### Expected behavior

Animated GIFs should loop endlessly to adhere to time-honored Internet tradition, also it is easy to miss the beginnings especially if readme's are long (if this is intended behavior, auto-play should start when GIF is in view only)

**Comment by bbkx226** [comment, marked as answer]
It's common to use animated GIFs in README.md files on GitHub to provide visual aids or demonstrations. By default, animated GIFs should loop endlessly to adhere to time-honored internet tradition, but in some cases, the GIF may stop after one loop.

To ensure that your animated GIF loops endlessly, you can try the following:

1. Check the source of your GIF: Ensure that the GIF you are using is actually a looping animation. Some GIFs may be single-loop animations or may not loop at all.

2. Use a different tool to generate the GIF: If the source of your GIF is not the issue, try using a different tool to generate the GIF. Some tools may generate GIFs that do not loop correctly.

3. Add a loop count: You can also explicitly set the number of loops for the animated GIF using the "loopcount" attribute. For example, to set the loop count to infinity, you can use the following syntax:

```
<img src="your_gif.gif" alt="your_alternative_text" width="your_width" height="your_height" loop=infinite>
```

If you are still experiencing issues with your animated GIFs, it's possible that there may be a bug or issue with the way GitHub is handling the files. You can try reaching out to GitHub support for further assistance.

You are welcome to indicate whether my answer was helpful by marking it.

**Comment by adamlui** [comment]
@bbkx226 you are da man, the GIF turned out to be single-loop so I converted it to endless one!
